src/pages/billing/List/BillingProjectActions.test.ts

```diff
@@ -39,16 +38,15 @@ describe('BillingProjectActions', () => {
   const verifyEnabled = item => expect(item).not.toHaveAttribute('disabled')
   const deleteProjectMock = jest.fn(() => Promise.resolve())
   const projectName = 'testProject'
+  const propsWithNoWorkspacesInProject = {
```

Comment:
My initial thought was that the cast to WorkspaceWrapper was an indication of something wrong.

Then I saw all the things that would need to be added in order to avoid it, which are obviously unnecessary to test this component, and how much noise they added.